examples/Standalone.html

```diff
@@ -1,21 +1,25 @@
 <!DOCTYPE html>
-<!-- Invokes Web WorldWind without using the internet. Refers to only data stored locally on the web server. -->
-<!-- To deploy:
-        - Download and unzip http://worldwindserver.net/webworldwind/WebWorldWind.zip.
-        - Download http://worldwindserver.net/webworldwind/WebWorldWindStandaloneData.zip and
-          unzip it into the Web WorldWind top-level directory so that the "standalonedata" directory is a peer
-          of examples, src, apps and worldwind.js.
-        - Ensure that your web server is configured to send an HTTP content type of application/octet-stream
-          or application/bil16 for files with a .bil suffix. You can do this for Apache servers by adding bil
-          to the list of application-octet-stream suffixes in the mime.types file and then restarting the server.
-        - Use your browser to invoke this HTML file, the one you're now reading, via the web server. Do not attempt
-          to open it using the browser's File->Open menu. Use an http or https URL with the server address.
-      If deployment was successful then you should see a Web WorldWind globe in the browser. The globe
-      should have blue marble imagery all over it and greenish Landsat imagery over most of North America.
-      If you zoom into a mountainous region you should see the raised mountains. There are no elevations
-      included for the oceans, so they are all flat. -->
-<!-- Be sure that the "../standalonedata" directory exists (relative to this file) and that it
-     contains the Blue Marble imagery, the Landsat imagery and the DTED0 elevations in its Earth directory. -->
+<!--
+Invokes Web WorldWind without using the internet. Refers to only data stored locally on the web server.
+
+To deploy:
+- Download https://worldwind32.arc.nasa.gov/WebWorldWind-StandaloneData.tar.gz and unzip it into the top-level
```

Comment:
After downloading the standalone data and running Standalone.html, it appears that some Landsat data is missing. Example: standalonedata/Earth/Landsat/0/0/0_0.png 404 (Not Found) Let me know if you want the full list.